content/blog/announcing-kustomize-support/index.md

```diff
@@ -0,0 +1,340 @@
+---
+title: "Announcing Kustomize Support"
+date: 2020-07-06
+draft: true
+meta_desc: "Pulumi adds Kustomize support to their Kubernetes provider"
+meta_image: meta.png
+authors:
+    - levi-blackstone
+
+tags:
+    - kubernetes
+---
+
+Kubernetes is complex, and there are many ways to manage Kubernetes resources. Pulumi supports many of these options, including native code SDKs, YAML, Helm, and now, Kustomize.
+
+<!--more-->
+
+Over the past year, we've heard from many users that they'd like to manage Kustomize resources directly with Pulumi.
```

Comment:
Move this before the break?

Reply:
I expanded the text on both sections a bit.